.npmignore

```diff
@@ -0,0 +1,8 @@
+*.swp
+*~
+*.iml
+.*.haste_cache.*
+.DS_Store
+.idea
+npm-debug.log
+lib
```

Comment:
Add `.babelrc` to this. We don't wanna send this to npm.
This may confuse a lot of things specially when using this inside a webpack based app.

And send me a PR to npm-base as well.